ibl-frontend/frontend-content/src/app/session-list/session-list.component.ts

```diff
@@ -340,11 +341,26 @@ export class SessionListComponent implements OnInit, OnDestroy {
         switchMap(() => {
           this.isLoadingTable = true;
           if(this.sort.direction == ''){
-            this.sort.active = 'session_start_time';
+            this.sort.active = 'nprobe';
             this.sort.direction = 'desc'
           }
           let filter = Object.assign({}, this.session_filter_form.getRawValue());
-          let newFilter = JSON.stringify(filter)
+
+          if(this.hideMissingPlots){
+            filter["nplot"] = 1
```

Comment:
Let's always make sure to terminate statements with a `;`. Please double-check your other changes here too.

```suggestion
            filter["nplot"] = 1;
```

Reply:
This has been resolved in the latest commit!